src/subscriber.ts

```diff
@@ -129,6 +133,7 @@ export class Subscriber {
     private async _handleNewHeadSubscriptions(): Promise<void> {
       this.subscribe.producers && this._initProducerHandler();
       this.subscribe.offline && this._initSessionOfflineHandler();
+      this.subscribe.offline && await this._initOutOfActiveSetHandler();
```

Comment:
We are checking `this.subscribe.offline` twice here, we could do instead:
```
if (this.subscribe.offline) {
  this._initSessionOfflineHandler();
  await this._initOutOfActiveSetHandler();
}
```
wdyt?

Reply:
Agree, I was thinking if it worth or not to split the this.subscribe.offline check... it involves different flavors of the same topic ( the offline status of the validator ), therefore for now I left it as a single check. Wdyt ?